web/cobrands/sass/_mixins.scss

```diff
@@ -10,20 +22,18 @@ $direction: 'left' !default;
     $nav_background_colour: $nav_fg_hover !global;
 }
 
-@mixin button-size() {
-    display: inline-block;
-    margin: 0;
-    padding: 0.75em 1em;
+@mixin button-size($mt:$mt, $mr:$mr, $mb:$mb, $ml:$ml) {
+    display: inline-block !important; //Override inline display: none on #geolocation_link
+    margin: $mt $mr $mb $ml;
```

Comment:
Given you’ve (correctly!) used `flip()` for the padding, I wonder whether it should also be used here, for the margin?